internal/scorecard/alpha/scorecard.go

```diff
@@ -39,52 +44,46 @@ type Scorecard struct {
 	bundleConfigMapName string
 	Client              kubernetes.Interface
 	SkipCleanup         bool
+	TestRunner          TestRunner
+}
+
+type PodTestRunner struct {
+	TestConfig Scorecard
+}
+
+type FakePodTestRunner struct {
```

Comment:
This is just a fake `TestRunner` right? Nothing to do with Pods?

```suggestion
type FakeTestRunner struct {
```

Reply:
renamed as you suggested.